cmd/agent/api/agent/agent.go

```diff
@@ -62,7 +62,8 @@ func stopAgent(w http.ResponseWriter, r *http.Request) {
 	w.Write(j)
 }
 
-func getVersion(w http.ResponseWriter, r *http.Request) {
+// GetVersion returns the version of the agent
+func GetVersion(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this should be extracted out of `api/agent/agent.go`, in a separate package, so that it's obvious that it's not only used in the `agent` package